torch/fx/experimental/proxy_tensor.py

```diff
@@ -82,29 +92,35 @@ def unwrap_proxy(e):
         args[0].proxy = proxy_out
         proxy_out.node.meta['tensor_meta'] = _extract_tensor_metadata(args[0])
 
-    with no_dispatch():
-        real_out = func_overload(*args, **kwargs)
-
+    real_out = func_overload(*pytree.tree_map(unwrap_elem, args), **pytree.tree_map(unwrap_elem, kwargs))
+    # Seems like func isn't marked as inplace_view for some reason?
+    if torch.Tag.inplace_view in func_overload.tags:  # type: ignore[attr-defined]
```

Comment:
nit: could you add a comment about syncing up the properties of the fake tensor/proxy tensor